.github/workflows/rosa-run-benchmark.yml

```diff
@@ -0,0 +1,90 @@
+name: ROSA Cluster - Run Benchmark
+
+on:
+  workflow_dispatch:
+
+concurrency: shared_cluster
```

Comment:
mostly a question for my clarity, this is to force only 1 job for all `shared_cluster` jobs ? where would this value `shared_cluster` be consumed ?

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency

Basically it prevents more than one workflow run with the same concurrency to be executed in parallel by GH Actions. That will even span multiple workflows, say if we introduce a separate "rosa-run-failure-testing.yml". We can and probably should also add these to the rosa-cluster-create/delete, as that would actually allow you to run cluster-create first, then run this workflow before even waiting for the cluster to be started.